frontend/api/auth/index.ts

```diff
@@ -0,0 +1,217 @@
+import passport from 'passport';
+import { sign } from 'jsonwebtoken';
+import express, { Express, Request, Response, NextFunction } from 'express';
+import { Strategy as FacebookStrategy } from 'passport-facebook';
+import { Strategy as GoogleStrategy } from 'passport-google-oauth20';
+import { Strategy as TwitterStrategy } from 'passport-twitter';
+import type { Profile } from 'passport';
+import { prisma } from '../../src/server/prisma';
+import { assignAnswerToUser } from '../answers';
+import ms from 'ms';
+import { respond404 } from '../../src/server/errors';
+
+const app: Express = express();
+
+if (process.env.VERCEL_ENV === 'preview') {
+  process.env[
+    'BASE_URL'
```

Comment:
Držme `PUBLIC_URL`, to už je definované.

```suggestion
    'PUBLIC_URL'
```